docs/livemedia-creator.rst

```diff
@@ -68,46 +69,47 @@ This is usually a good idea when testing changes to the kickstart. lmc tries
-and installs the system based on the kickstart. The ``%post`` section of the
-kickstart is used to customize the installed system in the same way that
-current spin-kickstarts do.
+The virtual machine (or host system if ``--no-virt`` is used) runs Anaconda
+with the Kickstart file provided to lmc by the ``--ks`` option to create a new
+Fedora installation in a filesystem image. The ``%post`` section of the Kickstart
+is used to customize the installed system. The filesystem image is created from
+scratch by Anaconda within the virtual machine; it is not based upon the ISO image
+passed to lmc with the ``--iso`` option.
 
 livemedia-creator monitors the install process for problems by watching the
 install logs. They are written to the current directory or to the base
 directory specified by the --logfile command. You can also monitor the install
-by using a vnc client. This is recommended when first modifying a kickstart,
+by using a vnc client. This is recommended when first modifying a Kickstart,
 since there are still places where Anaconda may get stuck without the log
 monitor catching it.
 
-The output from this process is a partitioned disk image. kpartx can be used
-to mount and examine it when there is a problem with the install. It can also
-be booted using kvm.
+If specified by ``part`` commands in the Kickstart file, the output from the Anaconda
+installation can be a partitioned disk image. kpartx can be used to mount and examine
+it when there is a problem with the install. It can also be booted using kvm.
 
-When creating an iso the disk image's / partition is copied into a formatted
-filesystem image which is then used as the input to lorax for creation of the
-final media.
+To create the final image, the filesystem generated by Anaconda within the
+virtual machine is used as the input to lorax. Lorax uses the templates in
+/usr/share/lorax/live/ or the live directory below the directory specified by
+``--lorax-templates``. The templates are written using the Mako template system
+with some extra commands added by lorax.
 
-The final image is created by lorax, using the templates in /usr/share/lorax/live/
-or the live directory below the directory specified by ``--lorax-templates``. The
-templates are written using the Mako template system with some extra commands
-added by lorax.
+If you want to stop before creating the final image from the filesystem installed
+by Anaconda, use the ``--image-only`` argument with lmc. To skip the install stage and use
+a previously created disk image, use ``--disk-image`` or ``--fs-image``.
 
 .. note::
-    The output from --make-iso includes the artifacts used to create the boot.iso;
-    the kernel, initrd, the squashfs filesystem, etc. If you only want the
-    boot.iso you can pass ``--iso-only`` and the other files will be removed. You
-    can also name the iso by using ``--iso-name my-live.iso``.
-
+    By default, lmc outputs the final image and the files used to create it
```

Comment:
This should keep the mention of --make-iso, other --make-* options have different results.